core/extension/gdextension_interface.h

```diff
@@ -1383,6 +1383,18 @@ typedef GDExtensionVariantFromTypeConstructorFunc (*GDExtensionInterfaceGetVaria
  */
 typedef GDExtensionTypeFromVariantConstructorFunc (*GDExtensionInterfaceGetVariantToTypeConstructor)(GDExtensionVariantType p_type);
 
+/**
+ * @name variant_get_internal_ptr
+ * @since 4.4
+ *
+ * Gets a pointer to the variant's internal value.
+ *
+ * @param variant The Variant.
```

Comment:
This needs to be updated to match:

```suggestion
 * @param p_variant The Variant.
```

Reply:
Whoops, thanks for being attentive!